script/Arcueid/Day 5/05_00_ARC05_1.txt

```diff
@@ -299,12 +299,12 @@ Not a single victim has been discovered―――?
 [sha:9945f275c190c472f276e6b4aecace1a00ee64ce]{
 -- Page 25, Offset 8650.
 -- なぜそこまで解っていてはっきりと口にしない。
-How can you say that when you have so much information?
+How can you say that when it's already so clear what happened?
 }
 [sha:c631ecbf4517c34f730ddf33ccbea27070809a1b]{
 -- Page 25, Offset 8651.
 -- ホテルに泊まっていた人間はみんな、あのワケのわからない炎にまかれて、跡形もなく燃え尽きてしまった事を。
-The information that all the people that stayed at the hotel were engulfed in those unfathomable flames and burned without a trace.
+Everyone who stayed in the hotel was engulfed in those unfathomable flames, reduced to cinder without leaving a single trace.
```

Comment:
I feel like staying _at_ a hotel is more common, but need to check ngeams. I also feel like reduced to ashes is more common than reduced to cinder. Has someone been playing too much fromsoft?